src/main/plugin/iso19115-3.2018/loc/eng/labels.xml

```diff
@@ -2153,7 +2153,8 @@
     <label>Classification system</label>
     <description>Name of the classification system</description>
   </element>
-  <element name="mri:cloudCoverPercentage" id="248.0">
+<!--  <element name="mri:cloudCoverPercentage" id="248.0"> -->
```

Comment:
Can you drop this line which is useless.

Reply:
sure I can